assets/js/googlesitekit/api/index.js

```diff
@@ -175,6 +175,12 @@ export const siteKitRequest = async (
 
 		return response;
 	} catch ( error ) {
+		if ( signal?.aborted ) {
+			// If the request was canceled, don't do any
```

Comment:
I can spot British spelling from a mile away 😉 

```suggestion
			// If the request was cancelled, don't do any
```

Reply:
We prefer the American spelling though, correct? 🤔 
![image](https://user-images.githubusercontent.com/1621608/150118305-887cf87a-a00f-4b1b-b60b-2dc05ff4da15.png)
https://www.merriam-webster.com/words-at-play/canceled-or-cancelled

Reply:
ARGH, oh no, right. This is the result of being Canadian; we mix both usages. And now I live in Europe. I have no clue how to English anymore 😅

Reply:
😄 if it makes you feel any better, the single `l` feels weird to me too 🤷‍♂️ 😆

Reply:
It does! 😅